src/slider.jsx

```diff
@@ -94,7 +93,7 @@ let Slider = React.createClass({
   },
 
   componentWillReceiveProps(nextProps) {
-    if (nextProps.value !== null) {
+    if (!nextProps.value) {
```

Comment:
I don't see a problem with your fix. This was probably a bug that's been in this component from the beginning.